### PR TITLE
Host JSON-LD context files

### PR DESCRIPTION
**Pull request by brechtvdv**

### CODE REVIEW

**Comment by brechtvdv** [comment]
Fixes https://github.com/StadGent/node_service_adlib-backend/issues/13

**Review by nvdk — APPROVED** [review]

**Comment by nvdk** [comment]
@aatauil did you mean to push to this PR?

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,12 @@
 {
-  "presets": ["@babel/preset-env"]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": 14
```
Comment (nvdk) [inline .babelrc:7]: I don't think this is necessary, but doesn't hurt either

**Comment by nvdk** [comment]
@brechtvdv perhaps for the  next PR it's better to create a dedicated branch ;).

**Pull request merged**